main.tex

```diff
@@ -843,41 +843,41 @@ \subsubsection{ASCII}
 % \subsubsection{\texttt{.ascii}: fast readers and writers}
 
 The \package{astropy.io.ascii} subpackage now includes a significantly faster
-Cython/C engine for reading and writing ASCII files. This is available for the
-most common formats.  On average the new engine is about 4 to 5 times faster
-than the corresponding pure-\python implementation, and is often comparable to
-the speed of the \package{Pandas} \citep{pandas} ASCII file interface.  The
+Cython/C engine for reading and writing ASCII files. This is available for most
+of the common formats.  On average, the new engine is about 4 to 5 times faster
+than the corresponding pure-\python implementation and is often comparable to
+the speed of the \package{pandas} \citep{pandas} ASCII file interface.  The
 fast reader has parallel processing option that allows harnessing multiple
 cores for input parsing to achieve even greater speed gains.  By default,
-\texttt{read()} and \texttt{write()} will attempt to use the fast C engine
+\texttt{read()} and \texttt{write()} will attempt to use the fast Cython/C engine
 when dealing with compatible formats. Certain features of the full read
-/ write interface are not available in the fast version, in which case the
+/ write interface are unavailable in the fast version, in which case the
 pure-\python version will automatically be used.
 
 % \subsubsection{\texttt{.ascii}: HTML tables}
 
 The \package{astropy.io.ascii} subpackage now provides the capability
-to read a table within an HTML file or web URL into an astropy
-\texttt{Table} object. Conversely a \texttt{Table} object can now
+to read a table within an HTML file or web URL into an \astropypkg
+\texttt{Table} object. Conversely, a \texttt{Table} object can now
```

Comment:
I know you didn't add this, but why is this sentence "Conversely"? Maybe just remove that word? "A \texttt{Table} object can now..."

Reply:
I think it was used as a "connector" for flow. I removed it and added "also".